src/Blaster.WebApi/Features/Shared/plugins/userManagementPlugin.js

```diff
@@ -0,0 +1,47 @@
+import Vue from "vue";
+import msal from 'vue-msal';
```

Comment:
![blaster-pr-broken-00](https://user-images.githubusercontent.com/1633308/68118973-b3e9b980-ff01-11e9-9fa9-55ecbe70636d.png)

Not sure if it's the wrapper or the msal library itself, but every page in Blaster that this is on, is broken.

Reply:
The problem is the vue-msal plugins usage of shorthand annotations, which isnt supported in IE11. I have been fighting with Babel to get it to include a single node_module to transpile, but it will require updates of other npm packages, which might lead us down the rabbit hole. So I have opted for posting an issue on the GitHub repo of the plugin author and hopefully he will fix this in a matter of days. If not we can fork the component and fix it ourselves. :)

Reply:
https://github.com/mvertopoulos/vue-msal/issues/5